src/store/modules/picklist/mutation-types.ts

```diff
@@ -1,3 +1,5 @@
 export const SN_PICKLIST = 'picklist'
 export const PICKLISTS_UPDATED = SN_PICKLIST + '/UPDATED'
 export const PICKLIST_CURRENT_UPDATED = SN_PICKLIST + '/CURRENT_UPDATED'
+export const PICKLIST_COMPLETED = SN_PICKLIST + '/COMPLETED'
```

Comment:
I think we could create a single mutation and add completed picklist in action, similarly we could reset items in actions and use same mutation 
```suggestion
export const PICKLIST_CMPLTD_UPDATED = SN_PICKLIST + '/CMPLTD_UPDATED'
```

Reply:
Also, we should store it with respect to logged in user. As there may be chances the logged in user is different. WDYT?

Reply:
Sir, Created a single mutation for adding the newly completed picklist and clearing the completed picklist.
Also, Sir, as if now we are having a single completed picklist for all the facilities of a single user, and clearing it once the user is logged out.
Thank you sir for reviewing my code.